pkg/odo/cli/component/create.go

```diff
@@ -345,42 +345,45 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 	// this populates the LocalConfigInfo as well
 	co.Context = genericclioptions.NewContextCreatingAppIfNeeded(cmd)
 
-	// If not using --s2i
-	if !co.forceS2i {
+	err = co.checkConflictingFlags()
+	if err != nil {
+		return
```

Comment:
return... what? This should be returning `err`

Reply:
Its defined here. https://github.com/openshift/odo/blob/ce72fb36681631b908ad7ca4731b2c4623db9823/pkg/odo/cli/component/create.go#L345 

That's a golang feature you can define the variable in the return block of function definition